src/style.css

```diff
@@ -1,3 +1,66 @@
+:root {
+  --primary-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+header {
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+  padding: 10px 50px;
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+header img {
```

Comment:
you should also make sure that `img` is positioned in the correct way (check mockup too):
![image](https://github.com/mate-academy/layout_moyo-header/assets/86923089/fc340343-8b58-4e4a-b778-b895454e3f67)